### PR TITLE
AUD-1736 Fix search behavior

### PR DESCRIPTION
**Pull request by balderdash**
Fixes: https://auderenow.atlassian.net/browse/AUD-1736

Make search actually case-insensitive (it previously never matched a search term if it contained capital letters), and split terms using spaces instead of commas.

**Reviewer should merge**
* No

### CODE REVIEW

File: src/util/search.ts [hunk 1]
```diff
@@ -32,25 +34,18 @@ export const containsSearchTerm = (
...
       }
     } else {
-      const defaultFilters = entry.hasOwnProperty("claimedCost")
-        ? DEFAULT_FILTERS_ENTRY
-        : DEFAULT_FILTERS_HISTORY;
       if (
-        defaultFilters.some(filter => {
+        DEFAULT_FILTERS_ENTRY.some(filter => {
+          if (entry[filter]) {
+          }
```
Comment (jcmorrow) [inline src/util/search.ts:45]: What's going on here?

File: src/util/search.ts [hunk 2]
```diff
@@ -32,25 +34,18 @@ export const containsSearchTerm = (
       const value = keyValue[1].toLowerCase().trim();
       if (
         entry[key] &&
-        entry[key]
-          .toString()
-          .toLowerCase()
-          .includes(value)
+        entry[key].toString().toLowerCase().includes(value.toLowerCase())
```
Comment (jcmorrow) [inline src/util/search.ts:37]: Do we say on the UI that searching by a specific field is an option? This seems really useful.

File: src/util/search.ts [hunk 3]
```diff
@@ -5,8 +5,16 @@ export interface DateRange {
   endDate: Moment | null;
 }
 
-const DEFAULT_FILTERS_ENTRY = ["patient", "item", "pharmacy"];
-const DEFAULT_FILTERS_HISTORY = ["id", "time", "description", "notes"];
+const DEFAULT_FILTERS_ENTRY = [
+  "patientFirstName",
+  "patientLastName",
+  "item",
```
Comment (jcmorrow) [inline src/util/search.ts:11]: Is "item" like sold product?

**Review by jcmorrow — APPROVED** [review]
A couple of small questions, but overall this looks good 🔍

File: src/util/search.ts (hunk 3 above)
Comment (balderdash) [reply src/util/search.ts]: That's right.

File: src/util/search.ts (hunk 2 above)
Comment (balderdash) [reply src/util/search.ts]: It shows up in the tooltip when you expand the search option in the pharmacy list, but yeah it's not super discoverable.

File: src/util/search.ts (hunk 1 above)
Comment (balderdash) [reply src/util/search.ts]: This is the "search without specifying a key" logic, i.e. look through all the available keys for a match. I just cleaned this up a bit.

**Pull request merged**